.circleci/config.yml

```diff
@@ -72,7 +72,8 @@ jobs:
           from: packages/IconFont/fonts
           to: s3://welcome-ui/public/fonts/$BRANCH/$ICON_FONT_HASH/
           arguments: |
-            --metadata GitCommit=$CIRCLE_SHA1 --delete
+            --metadata GitCommit=$CIRCLE_SHA1 --delete \
+            --cache-control "max-age=2678400"
```

Comment:
I think we also have a script `webfont:deploy` that also uses aws cli to upload font files. 
Since it is already handled by Ci, maybe we can just remove webfont:deploy from `release` script

Reply:
are you referring to this script: https://github.com/WTTJ/welcome-ui/blob/master/scripts/webfont-deploy.sh ?
I don't get where it is used exactly. Maybe @anucreative can tell us more about it since he wrote it.

Reply:
yes it's this one. when i talk about scripts, its about npm scripts located here: https://github.com/WTTJ/welcome-ui/blob/master/package.json#L7

Reply:
Summary of our meeting w/ @maxd-wttj, we should remove the call to `webfont:deploy` when executing a `release` or `release:prerelease` to let the ownership to the CI on how to deploy the webfont w/ wich options.
@anucreative is that ok for you ?

Reply:
Yeah, that makes sense. The idea is that CI takes care of releases when we merge into master ?

Reply:
yes, thats it !

Reply:
Hi guys!
I also think it's best to keep the webfont deploy outside of your release script. Should we merge this PR now? It will improve loading time on pages that use this asset.

Reply:
I'm just adding a commit to this PR to remove the duplication !

Reply:
finally created this PR to have separated reviews: https://github.com/WTTJ/welcome-ui/pull/1001

Reply:
On peut le merger maintenant n'est-ce pas @guillaumewttj